tslint.json

```diff
@@ -2,7 +2,6 @@
     "rules": {
         "no-unused-expression": true,
         "no-unused-variable": true,
-        "no-unreachable": true,

```

Comment:
Unreachable code is a on-by-default build breaker in tsc, controlled by [--allowUnreachableCode](https://www.typescriptlang.org/docs/handbook/compiler-options.html). We are not specifying a value for this parameter, so it defaults to false.

The tslint rule was deprecated, requiring that I remove it.